docs/rules/object-property-newline.md

```diff
@@ -72,20 +71,53 @@ Another benefit of this rule is specificity of diffs when a property is changed:
 +var obj = { foo: "foo", bar: "bazz", baz: "baz" };
 ```
 
-### Optional Exception
+### Optional Exceptions
+
+The rule offers three object option.
```

Comment:
Typo: option --> options

Reply:
Thank you, @platinumazure, for the review and suggestions. I’ll attend to these.